src/Workspaces/CSharp/Portable/Extensions/SyntaxTriviaExtensions.cs

```diff
@@ -32,12 +32,12 @@ public static bool MatchesKind(this SyntaxTrivia trivia, params SyntaxKind[] kin
 
         public static bool IsRegularComment(this SyntaxTrivia trivia)
         {
-            return trivia.IsSingleLineComment() || trivia.IsMultiLineComment();
+            return trivia.IsSingleLineComment() || trivia.IsMultiLineComment() || trivia.IsShebangComment();
```

Comment:
I wouldn't call it a ShebangComment.  It's not a comment.  It's just "IsShebang"